wd_helper.au3

```diff
@@ -272,7 +272,7 @@ EndFunc
 
 ; #FUNCTION# ====================================================================================================================
 ; Name ..........: _WD_WaitElement
-; Description ...: Wait for a element to be found  in the current tab before returning
+; Description ...: Wait for a element to be found in the current tab before returning

```

Comment:
This will need to be modified since you are adding a "not found" option.

Reply:
ok